test_honor_constrainOrbits test bad

I commited a minor documentation change to master which should have no effect on the code but the push fails unit testing in test_honor_constrainOrbits. The unit test output is shown below.

`FAIL: test_honor_constrainOrbits (tests.PlanetPopulation.test_PlanetPopulation.TestPlanetPopulation)

Traceback (most recent call last):
  File "/home/travis/build/dsavransky/EXOSIMS/tests/PlanetPopulation/test_PlanetPopulation.py", line 122, in test_honor_constrainOrbits
    self.assertTrue(np.all(a*(1-e) >= obj.rrange[0]),'constrainOribts low bound failed for %s'%mod.__name__)
AssertionError: constrainOribts low bound failed for AlbedoByRadius

Ran 208 tests in 1564.989s
FAILED (failures=1, skipped=3)
The command "coverage run -m unittest discover -v" exited with 1.
Done. Your build exited with 1.`

--- a/EXOSIMS/PlanetPopulation/AlbedoByRadius.py
+++ b/EXOSIMS/PlanetPopulation/AlbedoByRadius.py
@@ -96,7 +96,7 @@ class AlbedoByRadius(SAG13):
             sma = np.clip(a.to('AU').value, arcon[0], arcon[1])
             # upper limit for eccentricity given sma
             elim = np.zeros(len(sma))
-            amean = np.mean(arcon)
+            amean = np.mean(ar)
             elim[sma <= amean] = 1. - ar[0]/sma[sma <= amean]
             elim[sma > amean] = ar[1]/sma[sma>amean] - 1.
             elim[elim > self.erange[1]] = self.erange[1]

--- a/EXOSIMS/PlanetPopulation/SAG13.py
+++ b/EXOSIMS/PlanetPopulation/SAG13.py
@@ -171,7 +171,7 @@ class SAG13(KeplerLike2):
             sma = np.clip(a.to('AU').value, arcon[0], arcon[1])
             # upper limit for eccentricity given sma
             elim = np.zeros(len(sma))
-            amean = np.mean(arcon)
+            amean = np.mean(ar)
             elim[sma <= amean] = 1. - ar[0]/sma[sma <= amean]
             elim[sma > amean] = ar[1]/sma[sma>amean] - 1.
             elim[elim > self.erange[1]] = self.erange[1]

--- a/tests/PlanetPopulation/test_PlanetPopulation.py
+++ b/tests/PlanetPopulation/test_PlanetPopulation.py
@@ -118,8 +118,8 @@ class TestPlanetPopulation(unittest.TestCase):
             if (mod.__name__ not in exclude_check) and ('gen_plan_params' in mod.__dict__):
                 x = 10000
                 a, e, p, Rp = obj.gen_plan_params(x)
-                self.assertTrue(np.all(a*(1+e) <= obj.rrange[1]),'constrainOribts high bound failed for %s'%mod.__name__)
-                self.assertTrue(np.all(a*(1-e) >= obj.rrange[0]),'constrainOribts low bound failed for %s'%mod.__name__)
+                self.assertTrue(np.all(a*(1+e) <= obj.rrange[1]),'constrainOrbits high bound failed for %s'%mod.__name__)
+                self.assertTrue(np.all(a*(1-e) >= obj.rrange[0]),'constrainOrbits low bound failed for %s'%mod.__name__)
 
 
     def test_honor_prange(self):

